src/univariate/continuous/noncentralbeta.jl

```diff
@@ -39,9 +39,6 @@ partype(::NoncentralBeta{T}) where {T} = T
 
 @_delegate_statsfuns NoncentralBeta nbeta α β λ
 
-# TODO: remove RFunctions dependency once NoncentralChisq has its removed
-@rand_rdist(NoncentralBeta)
-
 function rand(d::NoncentralBeta)
     β = d.β
     a = rand(NoncentralChisq(2d.α, β))
```

Comment:
Similarly, we need
```suggestion
    a = rand(rng, NoncentralChisq(2d.α, β))
```
here.

Reply:
And similarly in the line below, but it seems I can't add a suggestion there.